kobolds/biology.md

```diff
@@ -74,25 +83,77 @@ scratching due to the intense itching. It is common for an individual to use
 tools and creams to ease the process and to engage the help of partners in
 scratching where it is hard to reach.
 
+### Reproductive Organs
 
-## Reproduction
-
-Kobold young are born from eggs. Eggs are internally fertilized during sex
-between two kobolds (in fact, there are some reports that fertilization may not
-always be required, but this is unconfirmed). Eggs are laid either one or two at
-a time only and a kobold capable of laying eggs may do so as few as zero or as
-many as twelve times in their life with one to three times being very common
+Kobolds' reproductive organs are between their legs, below their bellies and in
+front of the anus (which is near the underside of the base of their tails). Most
+of the time, all individuals' reproductive organs will look like a slit in the
+skin along the midline. During arousal or for urination, this slit expands and
+opens revealing, typically, either a vulva or a penis (though of course genetic
+variation accounts for less common arrangements). Fertilization is internal,
+occurring during sex between two kobolds. In fact, there are some reports that
+fertilization may not always be required to bear young, but this is unconfirmed.
+
+### Dewlap
+
+Kobolds have a dewlap on their chin and ventral side of their neck. This is
+often a bright, contrasting color from the rest of their body (e.g. a primarily
+green kobold may have a red or orange dewlap). Most of the time, the dewlap is
+tucked away along the chin and neck and cannot be seen. However, an individual
+can extend the dewlap a bit like a fan to display it. This is used in various
+social and cultural rituals, for instance related to finding sexual partners,
+intimidating foes, or other times of strong emotion.
+
+The dewlap can also change colors, which are generally visible even when the
+dewlap is relaxed. The color change is driven by the emotional state of the
+individual. It only responds to intense emotion and so does not ruin most
+kobolds' poker faces and reading the color changes is a subtle skill. Among
+individuals who do not know each other well, a rough understanding of each
+other's emotional state is possible, say "upset" or "happy". Whereas between
+individuals who have known each other for years, it may be possible to
+distinguish "embarrassed" from "frustrated" or "content" from "pleasantly
+surprised".
+
+
+## Physiology
```

Comment:
Carl said:
> I feel like the anatomy section is missing a description of their holistic physical nature. Can kobolds jump? How high? Climb? Swim? How long can they hold their breath? How many calories do they eat per day? How long can they go without protein? Do they cook their food? Do they ferment food? Do they eat salt? How long can they survive in freezing temperatures? How long can they survive in extreme heat? How dense are their bones? Can they see in the dark? How far can they see? How well can they smell?

On the topic of caloric intake, he mentioned the Japanese unit koku as a touch stone for ideas to avoid using the unit (kilo)calorie:
> One koku is the amount of rice one peasant family requires to survive during one year. Koku was a common currency for taxation in Sengoku times

Reply:
I'm making these questions into a task list so that I can remember which I've written answers to so far and which I still need to think about or run down.

[x] Can kobolds jump? How high?
[ ] Climb?
[x] Swim?
[x] How long can they hold their breath?
[x] How many calories do they eat per day?
[x] How long can they go without protein?
[ ] ~Do they cook their food? Do they ferment food?~ WONTFIX: These will go in sociology.
[ ] Do they eat salt?
[ ] How long can they survive in freezing temperatures?
[ ] How long can they survive in extreme heat?
[ ] How dense are their bones?
[ ] Can they see in the dark?
[ ] How far can they see?
[ ] How well can they smell?